city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,167 @@
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Request
+import datetime
+
+
+class ChiOhareNoiseSpider(CityScrapersSpider):
+    name = "chi_ohare_noise"
+    agency = "Chicago O'Hare Noise Compatibility Commission"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.oharenoise.org"]
+    meetings = dict()
+    start_urls = ["https://www.oharenoise.org/about-oncc/agendas-and-minutes"]
+    requests = []
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        table_rows = response.selector.xpath(".//tbody//tr")
+
+        for item in table_rows:
+            date = self._parse_start(item)
+            url_date = date.strftime("%Y/%m/%d")
+            meeting = Meeting(
+                title=self._parse_title(item),
+                links=self._parse_links(item),
+                start=self._parse_start(item),
+                source=self._parse_source(response)
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+            self.meetings[url_date] = meeting
+        curr_date = datetime.date.today()
+        months_urls = [datetime.date(month=curr_date.month-2, day=1, year=curr_date.year),
+                       datetime.date(month=curr_date.month-1, day=1, year=curr_date.year),
+                       curr_date,
+                       datetime.date(month=curr_date.month+1, day=1, year=curr_date.year)
+                       ]
+        for date in months_urls:
+            yield Request(
```

Comment:
You might need to add the `dont_filter=True` `kwarg` here as well so that it doesn't do the normal duplicate filtering